.travis.yml

```diff
@@ -15,6 +15,19 @@ env:
   - TEST_SUITE=brakeman
 matrix:
   fast_finish: true
+  jobs:
+  - sudo: required
+    group: edge
+    virt: lxd
+    dist: bionic
+    arch: arm64-graviton2
+    rvm: 2.6.6
+    env:
+    - RUBY_GC_HEAP_GROWTH_MAX_SLOTS=300000
+    - RUBY_GC_HEAP_INIT_SLOTS=600000
+    - RUBY_GC_HEAP_GROWTH_FACTOR=1.25
```

Comment:
As an aside, it's been several years since we came up with these numbers and I believe these were for appliances, not CI.  I don't remember if anyone evaluated them for performance with CI.

Reply:
Wonder if it's best to just drop them entirely (even on the appliances)

Reply:
It's worth reviewing at some point especially since we have vastly different environments: container based CI for vastly different test suites, appliances, podified with pretty primitive tools that destroy performance with low CPU request not matching actual usage and cause excessive restarts with memory limits lower than actual usage.

We never really get around to evaluating it though.